pkl-executor/src/test/kotlin/org/pkl/executor/EmbeddedExecutorTest.kt

```diff
@@ -121,7 +118,7 @@ class EmbeddedExecutorTest {
             "${Release.current().version().withBuild(null).toString().replaceFirst("dev", "SNAPSHOT")}.jar")
       path.apply {
         if (!exists()) throw AssertionError("Missing test fixture. " +
-          "To fix this problem, run `./gradlew :pkl-executor:prepareTest`.")
+          "To fix this problem, run `./gradlew :pkl-config-java:build`.")
```

Comment:
This doesn't look right. `:pkl-executor:prepareTest` should be exactly what's needed. In the current build, this can be abbreviated to `gw prepTest`, but that's not what I'd use in the message.

Reply:
Ah, I changed this because somehow `prepareTest` wasn't building a new `pkl-config-java-all` jar. But, I just tested again and it's working, so 🤷 

Will change this line back.

> I can't tell if you want every Gradle build to prepare for running EmbeddedExecutorTest with IntelliJ. If you do, there's no point in keeping the system properties around

Yep, that's the intention.